lib/licenses.nix

```diff
@@ -1060,6 +1060,11 @@ in mkLicense lset) ({
     url = "https://qwt.sourceforge.io/qwtlicense.html";
   };
 
+  radiance = {
+    fullName = "The Radiance Software License, Version 2.0";
+    url = "https://github.com/LBNL-ETA/Radiance/blob/master/License.txt";
+  };
+
```

Comment:
I'm wondering whether this change warrants its own commit… I'm leaning towards yes, any arguments against that?

Reply:
Thanks for the feedback here! I'd also argue in favour of this. After all I'm not precisely sure about the properties of the license as I'm not good at reading custom licenses and understanding all the little legal details.